signed-requests/README.md

```diff
@@ -0,0 +1,69 @@
+# signedrequests
+
+SignedRequest is written in Go and uses only Go in built packages. This tool provides users with following features
+
+1. Generates Public and Private Keys using Ed25519 Algorirthm
+2. Generate a base64 format for Public Keys
+3. Generate a signed request url with signature
+
+
```

Comment:
Start the instructions from Git Clone and walk the user to success. See GCP-Data-Drive readme as an example.